src/browser.js

```diff
@@ -13,16 +12,17 @@ const plugin =
   createPlugin({
     deps: {theme: MuiThemeToken.optional, jss: JssToken.optional},
     provides,
-    middleware(_, muiService) {
+    middleware(_, muiService: MaterialUIServiceType) {
       return async (ctx, next) => {
         if (!ctx.element) return next();
 
         ctx.element = await addProviders(ctx, muiService);
 
         await next();
 
-        const el = __BROWSER__ && document.getElementById('__MUI_STYLES__');
-        if (el) el.remove();
+        const el: mixed =
```

Comment:
Not sure declaring this as `mixed` is necessary. I think you might only need to do:
```
if (__BROWSER__ && document)
```